integration/dkg/dkg_emulator_suite.go

```diff
@@ -398,6 +398,15 @@ func (s *DKGSuite) initEngines(node *node, ids flow.IdentityList) {
 	)
 	require.NoError(s.T(), err)
 
+	// We add a hook to the logger such that the test fails if the broker writes
```

Comment:
Nice, just making sure this hook is active only when testing for happy path cases. For unhappy paths, these warnings could happen and it shouldn't make the test fail.